### PR TITLE
fix: 修复当"远程控制"没有填入任何内容时的界面问题

### PR DESCRIPTION
**Pull request by hsyhhssyy**
image

现在如果配置项未填写，界面会是这样。

顺便加入了提示和文档链接。

### CODE REVIEW

**Review by MistEO — APPROVED** [review]

File: src/MaaWpfGui/Res/Localizations/en-us.xaml [hunk 1]
```diff
@@ -684,6 +684,10 @@ The video aspect ratio needs to be 16:9 without interference factors such as bla
     <system:String x:Key="RemoteControlConnectionTestFailNotHttps">Connection Test Failed. Endpoint is not an https address.</system:String>
     <system:String x:Key="RemoteControlReceivedTask">Received remote task {0}, Task Id:{1}.</system:String>
     <system:String x:Key="RemoteControlCompletedTask">Completed remote task {0}, Task Id:{1}.</system:String>
+    <system:String x:Key="RemoteControlTooltips">Caution: Entering addresses from unknown sources may result in loss to your Arknights account.</system:String>
```
Comment (Constrat) [inline src/MaaWpfGui/Res/Localizations/en-us.xaml:687]:
```xml
<system:String x:Key="RemoteControlTooltips">Caution: Entering addresses from unknown sources may result in the loss of your Arknights account.</system:String>
```

**Review by Constrat — CHANGES_REQUESTED** [review]

File: src/MaaWpfGui/Res/Localizations/en-us.xaml [hunk 2]
```diff
@@ -684,6 +684,10 @@ The video aspect ratio needs to be 16:9 without interference factors such as bla
...
+    <system:String x:Key="RemoteControlDeveloperTooltips">To learn about how to develop related features, visit</system:String>
```
Comment (Constrat) [inline src/MaaWpfGui/Res/Localizations/en-us.xaml:688]:
```xml
<system:String x:Key="RemoteControlDeveloperTooltips">To learn more about how to develop these features, visit</system:String>
```

**Review by Constrat — CHANGES_REQUESTED** [review]

**Comment by Constrat** [comment]
My bad, I've created to revies instead of 1

**Review by Constrat — APPROVED** [review]

**Comment by hsyhhssyy** [comment]
I have changed these en-us localization string content.

**Pull request merged**